### ISSUE TITLE
There's a broken link on the Accessibility page

### ISSUE THREAD

**Issue by kraccoon-dev** [mdn/content#17730]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility

### What specific section or headline is this issue about?

See also

### What information was incorrect, unhelpful, or incomplete?

A broken link was found in "See Also" on the page.

![image](https://user-images.githubusercontent.com/69508345/176144229-78acb78a-312d-4f75-823c-9975547cb32f.png)

### What did you expect to see?

It should be removed or changed to correct path

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/index.md
* Last commit: https://github.com/mdn/content/commit/019d7181d615c7ae0e1f07549674030e4f865d16
* Document last modified: 2022-05-08T05:34:10.000Z

</details>

**Comment by wbamberg** [comment]
I think we can just delete this link.
